app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -241,32 +238,45 @@ const AddEditNotificationModal = ({
     },
   ];
 
-  const {
-    actionType,
-    notification: { sendCase, informOwner },
-  } = data;
+  const { actionType } = data;
 
-  const okButton = {
-    text: formatMessage(COMMON_LOCALE_KEYS.SAVE),
-    onClick: () => {
-      const modalName = `${messages.title.defaultMessage.replace(
-        /\{.*\}/i,
-        messages[actionType].defaultMessage,
-      )}`;
+  const submitActions = (formFieldValues) => {
+    const modalName = `${messages.title.defaultMessage.replace(
+      /\{.*\}/i,
+      messages[actionType].defaultMessage,
+    )}`;
+
+    const newFormValues = { ...formFieldValues };
+
+    const {
+      informOwner,
+      sendCase,
+      enabled: switcher,
+      attributes: { length },
+    } = newFormValues;
 
-      handleSubmit(onSave(!isEditorShown))();
+    newFormValues.attributes = !isEditorShown ? [] : newFormValues.attributes;
```

Comment:
You can do it directly when define `newFormValues`
const newFormValues = { ...formFieldValues , attributes: !isEditorShown ? [] : formFieldValues.attributes };